makefile

```diff
@@ -361,6 +361,9 @@ gqlgen:
 	$(gobin) run github.com/99designs/gqlgen generate
 	$(gobin) run cmd/gqlgen/generate_secret_fields.go
 
+govul-install:
```

Comment:
For discoverability (some shells will autocomplete `make` targets, and people may read the `makefile` themselves), I would suggest adding a target to run `scripts/check-go-vulnerabilities.sh`, and calling that `make` target from `self-tests.yml`.

Reply:
I'm not convinced about adding a target for it since to run it locally, it would be the same as all the other verify scripts. The other verify scripts also do not have a target (only one does and it's instead a go command rather than a shell script). I think I should either add all the scripts to the makefile and rewrite the self-tests to use those targets or write none to keep consistency. I'm leaning towards not because it adds length to the makefile.

Alternatively I thought about using a wildcard syntax in the makefile since most follow a "verify-" and then adding only a couple commands but it wouldn't have great autocomplete then. What do you think?

Reply:
That (not adding to the makefile) sounds reasonable to me.